github/compliance/issue.py

```diff
@@ -81,6 +82,9 @@ def target_name(
         artifact = cnudie.iter.artifact_from_node(target)
         return f'{target.component.name}:{artifact.name}'
 
+    elif isinstance(target, cmr.CfgElementStatusReport):
+        return target.name
+
     else:
         raise TypeError(f'{type(target)} not a valid target')
```

Comment:
the exception-message is not quite accurate. most likely, this exception will be raised if we add (sometime in the future) another "target-type", and forget to add a codepath for it here. in which case the value would very well be valid, but not yet "supported".

at any rate: just raise TE w/ the offending value as sole parameter. always. be specific if and only if you are sure to always be correct. otherwise, be less specific. this will also save some typing